### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by jjb182** [arnaudelub/flutter_fullpdfview#9]
Hi there, I am wondering if you will be supporting a *Search Function* in up coming releases?

I think that this would be Immensely useful!

Thank you

**Comment by gitdamilare** [comment]
Yeah, I think that was great, and I just want to say thank you for the great work on this library. It really nice

**Comment by arnaudelub** [comment]
Hi, thanks for pointing it, i will work on it as soon as i can, but if you wish you can still do a PR with this enhancement :)

**Comment by jjb182** [comment]
If the PDF was searchable that would be FANTASTIC!!

I'm just about to release an app where that would be useful, and I'd love
to test it for you and implement it if you get it done.

**Comment by jjb182** [comment]
Hi there, I'm wondering if there has been any movement on this enhancement?

I'm not savvy enough for a PR, but I could pass you some code for the iOS
side that might be helpful.

I'm mostly interested in the Android side now.

Thanks!

**Comment by arnaudelub** [comment]
well, adding a search Engine is not an easy task, it's surely a nice enhancement but it may take some times
